sklearn/utils/validation.py

```diff
@@ -295,8 +295,10 @@ def check_array(array, accept_sparse=None, dtype="numeric", order=None,
         If dtype is a list of types, conversion on the first type is only
         performed if the dtype of the input is not in the list.
 
-    order : 'F', 'C' or None (default=None)
-        Whether an array will be forced to be fortran or c-style.
+    order : 'F', 'C' or 'A' (default='A')
```

Comment:
I had already raised and issue here
https://github.com/numpy/numpy/issues/7374
I don't remember exactly but some Cython code which was expecting a C-contiguous array was not getting it.